corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -726,6 +726,8 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
         self.parent = parent;
         Container.call(self, json);
 
+        self.rel_ix = ko.observable(relativeIndex(self.ix()));
```

Comment:
why was this needed?

Reply:
There was console error on clicking 'X' for a repeat group - `rel_ix is not a function`

Reply:
Was the error related to this [part](https://github.com/dimagi/commcare-hq/blob/60f2cfb1ca875b6383ca8f3f4f0a881cfdcc2924/corehq/apps/cloudcare/static/cloudcare/js/form_entry/web_form_session.js#L440)? I'm guessing you don't need this anymore after implementing this [commit](https://github.com/dimagi/commcare-hq/pull/33834/commits/80730cfcd989e15bf033d808ffbc08a5056c599f), can you give it a try?

What is the output of `relativeIndex` and what is it used for? I'm wondering mostly because I don't think it currently supports the `ix` values of `GroupedElementTileRow`. The `ix` value would look something like `(0,0_0,1)` and `relativeIndex` currently doesn't handle the paranthesis.

Reply:
Yes, it works now without having this statement. Removed it.